sdks/python/apache_beam/runners/interactive/README.md

```diff
@@ -27,38 +27,142 @@ exploration much faster and easier. It provides nice features including
 
 1.  Graphical representation
 
-    When a pipeline is executed on a Jupyter notebook, it instantly displays the
-    pipeline as a directed acyclic graph. Sampled PCollection results will be
-    added to the graph as the pipeline execution proceeds.
+    Pipeline DAG can be visualized through:
```

Comment:
Use a more active voice for the comments. For example this line could read "Visualize the Pipeline DAG:". And "Visualize elements in a PCollection:"

Reply:
Thanks! Made the amendments.